vscode-typescript/track.yml

```diff
@@ -0,0 +1,50 @@
+slug: vscode-typescript
+id: 0dxngfkws4op
+type: track
+title: VSCode & TypeScript
```

Comment:
I believe this template exists to demonstrate VSCode. Consider to make that clear in the title and remove Typescript - it's a distractor.

Reply:
Copy is not final yet.